lib/beacon/loader/page.ex

```diff
@@ -124,10 +125,10 @@ defmodule Beacon.Loader.Page do
   end
 
   defp handle_event(page) do
-    %{site: site, event_handlers: event_handlers} = page
+    event_handlers = Content.list_event_handlers(page.site)
```

Comment:
There's still one step missing here :)

Whenever an event handler changes we'd need to broadcast a message to recompile all page modules so I'm wondering if it would be better to create a new `EventHandler` module containing all `handle_event` and on the `handle_event` of `PageLive` here https://github.com/BeaconCMS/beacon/blob/ce916b5c5f20e94737f3f6b54e1356ae525d6be9/lib/beacon/web/live/page_live.ex#L70-L76

We just replace `page_module` with `event_handler_module`.

In this case whenever an event handler changes we only recompile the single event handler module and save resources by not duplicating the same event_handler on all page modules.

Wdyt?